distributed-testing/build.gradle

```diff
@@ -0,0 +1,63 @@
+apply plugin: 'groovy'
+apply plugin: 'java-gradle-plugin'
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    jcenter()
+}
+
+gradlePlugin {
+    plugins {
+        distributedTestingPlugin {
+            id = 'net.corda.plugins.distributed-testing'
+            implementationClass = 'net.corda.plugins.DistributedTesting'
+        }
+    }
+    automatedPublishing = false
+}
+
+
+pluginBundle {
+    website = project.vcsUrl
+    vcsUrl = project.vcsUrl
+    description = 'Corda Distributed Testing Plugin'
+    tags = ['corda']
+
+    plugins {
+        distributedTestingPlugin {
+            displayName = 'Corda Distributed Testing Plugin'
+        }
+    }
+}
+
+dependencies {
+    compile gradleApi()
+    compile "io.fabric8:kubernetes-client:4.4.1"
+    compile 'org.apache.commons:commons-compress:1.19'
+    compile 'org.apache.commons:commons-lang3:3.9'
+    compile 'commons-codec:commons-codec:1.13'
+    compile "io.github.classgraph:classgraph:4.8.41"
+    compile "com.bmuschko:gradle-docker-plugin:5.0.0"
+    compile 'org.apache.commons:commons-csv:1.1'
+    compile group: 'org.jetbrains', name: 'annotations', version: '13.0'
+    testCompile "junit:junit:4.12"
+    testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
+    testImplementation "org.junit.jupiter:junit-jupiter-api:$junit_jupiter_version"
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junit_jupiter_version"
+}
+
+test {
+    useJUnitPlatform()
+    afterSuite { suite, result ->
+        result.exception?.printStackTrace()
+    }
+}
+
+allprojects {
```

Comment:
You don't need this `allprojects` block because the top-level `build.gradle` does this already.